### ISSUE TITLE
I've have been wrongly banned.

### ISSUE THREAD

**Issue by MrWxlfz** [bloxstreet/support#15]
Hey there! I have been a loyal player to BloxStreet for years and was previously a Board of Directors member.

I took a LONG break from the game from late 2022 till now. But when I join it says this; "You have been **banned from BloxStreet** for reclaiming a role while blacklisted.. ..Appeal in our communications server". 

I was devastated that one of my favorite games which I spent a ton of Robux on was taken away from me. But the worst part is that when I try to appeal and **join the Discord, I have been banned**. So I can't even submit and appeal for a wrongful enforcement action. I have provided screen shots below.

Please help me, thanks.
![Screenshot 2024-11-03 143403](https://github.com/user-attachments/assets/f5e10e93-b23d-4556-a563-a625b26dab54)
![Screenshot 2024-11-03 143313](https://github.com/user-attachments/assets/48bad316-6277-4702-b0b3-65cc39f70dfb)

**Comment by cyus874y665** [comment]
i have benn wrongly banned firegamer580 banned me because i was in the elevator and i was doing a task and when i got back he banned me and my rank was gone

**Comment by avvlena** [comment]
Ask camerrrcncmonghan to unabn u

**Comment by avvlena** [comment]
Trust

**Comment by MrWxlfz** [comment]
@avvlena on discord?

**Comment by avvlena** [comment]
> [@avvlena](https://github.com/avvlena) on discord?

Yep. This is his username on discord i think, @camerxnmonaghxn hes a group director or somrthing, very kind person btw, just dont yell at him, hes kind of sensitive. Or if you want to try with another shr, ask uhh derrion or louie :D also wtf, devs should add a system to stop people from buying ranks when they are blacklisted. And i dont blame you, cuz you enter the game, and it doesnt even say you are blacklisted, you just lose your rank, but you could think it was just a demotion, a bug, not a blacklist. (Another thing that shows how bad this game was developed)  Like wtf you wasted 10k fuckng robux just so they would end up banning youm that, in my opinion, says a lot about the type of people they are. But they are so lazy i swear to god they wouldnt even add a cuff log system. But ofc they want ppl to spend their money buying their gamepasses to feed the little postlyticket. Being honest the bloxsteet devs and overall the bloxstreet community is so unambitious. Kind of lame. If you reached this part and you are reading this, i dont think @camerxnmonaghxn will unban you, and hes not kind like i said before (imo) but dm him anyway so he doesnt feel too alone. Anyways, bye. I hope u get unbanned

**Comment by avvlena** [comment]
@camerxnmonaghxn pls fix ur game 😭😭🥀

**Comment by camerxnmonaghxn** [comment]
Appeal your ban in the communications server.
